examples/examples.test.ts

```diff
@@ -12,8 +12,8 @@ const vgSchema = require('vega/build/vega-schema.json');
 
 const ajv = new Ajv({
   validateSchema: true,
-  extendRefs: 'fail',
-  allErrors: true
+  allErrors: true,
+  extendRefs: 'ignore'
```

Comment:
It's complicated. Bascially, JSON schema does not allow additional properties when you use a   `$ref`. However, we often use references and descriptions together. 

I have a hunch that that may actually be fine but you are not allowed to use a `$ref` and also define `@minimum` for example. Let me investigate this.